_content/overview.md

```diff
@@ -1,100 +1,103 @@
+```
 
 #### Correct
 
-Our content must be technically and grammatically correct.
+Our content must be factually and grammatically correct.
 
-Incorrect content causes our audience to lose trust in our content and brand. If you are unsure if your content is technically correct, check with a developer before publication.
+Incorrect content causes our audience to lose trust in our documentation and brand. If you are unsure if your content is correct, check with a developer before publication. //TODO William Is there not a technical review at some point?
 
-Grammatically incorrect writing confuses our readers and reflects poorly on our professionalism. If you are questioning the wording of your content, speak to another writer, or contact a technical editor for help.
+Grammatically incorrect writing can confuse our readers and reflects poorly on our professionalism. If you are having trouble with the wording of your content, speak to another writer, or contact a technical editor for help.
 
 ### General principles
 
 #### Style and tone
 
-Best practices to adopt:
-
-- Plan what to write before you write.
-- Always keep your audience in mind when you write.
-- Use active voice when possible.
-- Write for scannability; use bullet points, short paragraphs, and sections/headers to break up your content.
-- Be brief.
-- Oxford comma: In a list of three or more items, add a comma before the conjunction (for example: "Android, iOS, and Windows").
-- Spell check your content.
-- Be friendly by using "you". And remember that Agora can "recommend" courses of action.
-- Review your content. Edit out any information your reader does not need to know.
-- Use parallel structures to tell the reader how related concepts are different and how they are similar.
-- Place essential information in the main clause, as you cannot assume the reader will read the whole sentence.
-- Remove ambiguity by choosing words whose meaning is clear.
-- Use the relative pronoun "that" where you might otherwise omit it, and prefer using it to a noun-...ing combination, for example say "the method that uses this parameter" instead of "the method using this parameter".
+Best practices to adopt: //TODO William Are there too many? Should we simplify this or break it up? If you wanted something someone could maybe print (on one page) and put up by their workstation, this might be too much.
+
+* Create an outline to plan what to write before you start.
+* Always keep your audience's capabilites and needs in mind when you write.
+* Write for scannability by using bullet points, short paragraphs, and informative section headings to break up your content.
+* Keep paragraphs short. Dense pages full of text can be intimidating for readers (or hide important information from skimmers).
+* Remember to be clear and concise. Replace phrases with single words where possible, and remove ambiguities.
```

Comment:
This one is repeating the previous sections.

Reply:
I think there will be a lot of deletion and compression in this section. I have seen a few good examples of short "cheat sheets".